README.md

```diff
@@ -52,36 +52,48 @@ $ git commit -m "DVC init"
 
 ## Example code
 
-Copy the snippet below as a basic example of the API usage:
+Copy the snippet below into `train.py` for a basic API usage example:
 
 ```python
-# train.py
+import time
 import random
-import sys
+
 from dvclive import Live
 
+params = {"learning_rate": 0.002, "optimizer": "Adam", "epochs": 20}
```

Comment:
I don't love that params don't have an effect, but not a blocker

Reply:
Yes, this is a for a fake feel of familiarity / realism. But I've seen this in other places and I think since this is very very explicitly "fake" it's not misleading anyone (personal feel)